source/blog/2016-12-06-part1-install-origin-on-f25-atomic-host.html.md

```diff
@@ -0,0 +1,357 @@
+    State: idle
+    Deployments:
+    ● fedora-atomic:fedora-atomic/25/x86_64/docker-host
+           Version: 25.42 (2016-11-16 10:26:30)
+            Commit: c91f4c671a6a1f6770a0f186398f256abf40b2a91562bb2880285df4f574cde4
+            OSName: fedora-atomic
+
+Looks like they are up and all at the same state. The next step is to
+unleash the installer. Before we do we should note that Fedora has moved
+to python3 by default. While Atomic Host still has python2 installed
+for some reasons not all of the modules needed by the installer are
+there in python2 form. Thus, we'll forge ahead and use python3 as the
+interpreter for ansible by specifying
+`-e 'ansible_python_interpreter=/usr/bin/python3'` on the command line:
+
+    $ ansible-playbook -i myinventory playbooks/byo/config.yml -e 'ansible_python_interpreter=/usr/bin/python3'
+    Using /etc/ansible/ansible.cfg as config file
+    ....
+    ....
+    PLAY RECAP *********************************************************************
+    52.91.115.81               : ok=162  changed=49   unreachable=0    failed=0   
+    54.175.0.44                : ok=540  changed=150  unreachable=0    failed=0   
+    54.204.208.138             : ok=159  changed=49   unreachable=0    failed=0   
+    localhost                  : ok=15   changed=9    unreachable=0    failed=0
+
+We snipped pretty much all of the output. You can see the log in its
+entirety [here](/images/2016-12-05_origin-install-output.txt.gz).
+
+So now the installer has run and our systems should be up and running.
+There is only one more thing we have to do before we can take this
+system for a spin.
+
+We created two users `user` and `admin`. Currently there is no way to
+have the installer associate one of these users with the *cluster admin*
+role in OpenShift (we opened a
+[request](https://github.com/openshift/openshift-ansible/issues/2877)
+for that). We must run a command to associate the `admin` user we
+created with cluster admin role for the cluster. The command is
+`oadm policy add-cluster-role-to-user cluster-admin admin`.
+
+We'll go ahead and run that command now on the master node via
+`ansible`:
+
+    $ ansible -i myinventory masters -a '/usr/local/bin/oadm policy add-cluster-role-to-user cluster-admin admin'
+    54.175.0.44 | SUCCESS | rc=0 >>
+
+And now we are ready to log in as either the `admin` or `user` users
+using `oc login https://54.175.0.44:8443` from the command line or
+visiting the web frontend at `https://54.175.0.44:8443`.
+
+To Be Continued
+===============
+
+In this blog we brought up an OpenShift Origin cluster on three servers
+that were running Fedora 25 Atomic Host. We reviewed the inventory file
+in detail to explain exactly what options were used and why. In a future
+blog post we'll take the system for a spin and inspect some of the
+running system that was generated from the installer and also spin up an
+application that will run on and be hosted by the Origin cluster.
+
```

Comment:
We will also add a more generalized form of these instructions to the main ProjectAtomic and Fedora Atomic Host documentation.